docs/terraform-integration-tests.md

```diff
@@ -0,0 +1,62 @@
+DC/OS is tested in TeamCity by several groups of integration tests using dcos-terraform and AWS spot instances.
+
+# Connecting to a test cluster
+
+Once the TeamCity job has completed, the DC/OS cluster that was created will be deleted. However, if the job is still running, you can connect to the cluster.
+
+Once `terraform apply` has completed, the job outputs to the job log the generated SSH key, cluster address, and node IP addresses as well as a bash one liner to write the SSH key to disk and SSH into the master node.
```

Comment:
There's no reason to include this information in the job artifacts as it will all be deleted by the time it gets there, so it's outputted to the job log.

Reply:
As long as its there and people know where to find it: no need to have it in the artifacts.